src/components/Footer/footer.module.css

```diff
@@ -0,0 +1,42 @@
+.footer {
+  background-color: var(--background-gray);
+  height: 27.7vw;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.bottomLinks {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.topLinksBar {
+  margin-top: 3.7vw;
+  margin-bottom: 3.7vw;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: center;
+}
+
+.topLink {
+  margin-right: 6.66vw !important;
```

Comment:
You know that important is one of the worst practices. Why you keep using bad practices like position absolute or important? This kind of regress makes me sad 😭, it was emphasized on one of the first lessons not to use it.

Reply:
done